frontend/src/store/modules/app.js

```diff
@@ -133,13 +134,14 @@ export const useAppStore = defineStore("app", {
           await this.getBatchJobTypeCodes();
           await this.getTranscriptTypeCodes();
           await this.getCertificateTypeCodes();
+          await this.getStudentGradeCodes();
           // GET & SET INSTITUTE SCHOOL AND DISTRICT LISTS
           await this.getSchools();
           await this.getDistricts();
           // GET & SET INSTITUTE CODES
           await this.getInstituteCategoryCodes();
           await this.getInstituteFacilityCodes();
-          await this.getInstituteGradeCodes();
+          await this.getInstituteGradeCodes();        

```

Comment:
Can you remove the institute grade codes getters, setters, and data from the app store since it's not in use?